### PR TITLE
Add Eclipse

### PR DESCRIPTION
**Pull request by ezzzin**
**Issue:#1097

### Checklist
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by ezzzin** [comment]
Thanks for letting me know! I modified it and pushed it back.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -560,6 +560,11 @@
             "hex": "E53238",
             "source": "https://go.developer.ebay.com/logos"
         },
+        {
+            "title": "Eclipse",
+            "hex": "F7941E",
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The color associated with the Eclipse IDE seems to be the purple `#2C2255` (taken from the text in the Eclipse IDE SVG), with the orange `#F7941E` being primarily for accents. The Eclipse Foundation uses the orange as their primary color.

File: icons/eclipse.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...042969 23.785156 9.988281 23.761719 9.988281 C 23.742188 9.988281 18.988281 9.988281 13.195312 9.992188 Z M 13.195312 9.992188 "/><path d="M 2.53125 12.648438 C 2.511719 12.6875 2.554688 13.203125 2.605469 13.617188 C 2.625 13.773438 2.640625 13.914062 2.640625 13.929688 C 2.640625 13.957031 4.769531 13.964844 13.222656 13.964844 L 23.804688 13.964844 L 23.828125 13.777344 C 23.871094 13.453125 23.929688 12.8125 23.921875 12.714844 L 23.914062 12.621094 L 13.230469 12.613281 C 3.898438 12.609375 2.542969 12.613281 2.53125 12.648438 Z M 2.53125 12.648438 "/><path d="M 2.953125 15.355469 C 2.953125 15.375 3.007812 15.5625 3.074219 15.773438 C 3.519531 17.148438 4.191406 18.359375 5.136719 19.5 C 5.417969 19.839844 6.078125 20.53125 6.359375 20.78125 C 7.921875 22.175781 9.59375 22.988281 11.519531 23.300781 C 12.121094 23.394531 12.402344 23.414062 13.261719 23.410156 C 14.167969 23.410156 14.554688 23.378906 15.277344 23.234375 C 17.378906 22.820312 19.175781 21.789062 20.785156 20.078125 C 22.046875 18.734375 22.882812 17.328125 23.402344 15.683594 L 23.519531 15.320312 L 13.234375 15.320312 C 4.386719 15.320312 2.953125 15.324219 2.953125 15.355469 Z M 2.953125 15.355469 "/></svg>
```
Comment (birjolaxew) [inline icons/eclipse.svg]: :+1: Nice work on the SVG! Could you give it a run through [SVGOMG](https://jakearchibald.github.io/svgomg/) to see if you can lower the precision a bit without introducing artifacts? I usually find a precision of 3 to be the sweet spot. Also remember to update the `<title>` when you update it in the JSON :)

**Comment by gabrieledarrigo** [comment]
Ciao guys,
glad to help on this, so if it's not a problem I can step in and complete the work!
Let me know : )

**Comment by chris48s** [comment]
I'm also happy to cherry-pick the commits from this branch and take it over the line if its abandoned

**Comment by ericcornelissen** [comment]
At this point anyone is free to take over this PR in my opinion 👍

**Pull request closed without merging**